sdk/eventhub/Azure.Messaging.EventHubs/src/Compatibility/TrackOneGenericTokenProvider.cs

```diff
@@ -22,7 +22,7 @@ namespace Azure.Messaging.EventHubs.Compatibility
     internal sealed class TrackOneGenericTokenProvider : TokenProvider
     {
         /// <summary>The default scope to use for token aquisition with the Event Hubs service.</summary>
-        private static readonly string[] EventHubsDefaultScopes = new[] { "https://eventhubs.Azure.net//.default" };
+        private static readonly string[] EventHubsDefaultScopes = new[] { "https://eventhubs.azure.net/.default" };
```

Comment:
Ouch.   I feel like that must have been a copy/paste from somewhere but, it seems like that should have been an obvious catch on my part.  

Thanks!

Reply:
We should probably try to figure out why we didn't catch it earlier. This might mean we aren't doing any live tests with ```ClientSecretCredential``` otherwise the test would have failed - maybe it is already and we just haven't fixed it yet!

Reply:
It wasn't a well tested code path due to the non-goal of focusing and highlighting support for credential types outside of the connection string/shared access key for the first preview.   This was only included as a precursor until the `Azure.Identity` library was ready, at which point, this would have been tested and validated.